app/models/event.rb

```diff
@@ -0,0 +1,10 @@
+class Event < ApplicationRecord
+  belongs_to :user
```

Comment:
The user should be referenced as the event creator i.e. `belongs_to :creator, class_name: "User"` ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/associations/user_assoc_requirement.png) from [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/associations))